src/components/Image/Image.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+export function Image(props) {
+  const { src, alt } = props;
+
+  return (
+    <>
+      <img src={src} alt={alt} />
+    </>
+  );
+}
+
+Image.propTypes = {
+  /**
+   * The text that the button will display
+   */
+  id: PropTypes.string,
```

Comment:
Minor updates needed to the explanation text for each parameter, to explain the individual parameters usage :)